src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -43,10 +44,38 @@ class Select<T> extends React.Component<Props<T>> {
 
     @observable selectedOptionRef: ?ElementRef<'li'>;
 
+    @observable buttonRefs: Map<number, ElementRef<'button'>> = new Map();
+
+    @observable searchText: string = '';
+
+    @observable focusedElementIndex: number = -1;
+
     @observable open: boolean;
 
+    @computed get buttonTexts() {
+        return Array.from(this.buttonRefs.entries())
+            .map<[number, string]>(([index, ref]) => [index, ref.textContent]);
+    }
+
+    @computed get selectedIndex() {
```

Comment:
yes, but I think it's better than using the first item as first focused element, because if you have a very long select list (like countries) the current selected item is always centered. so if you would just press arrow down, the list would jump to the top

Reply:
thats true, but i think i would call this `firstSelectedIndex` then

Reply:
fixed